### PR TITLE
geOrchestra 24.0

### PR DESCRIPTION
**Pull request by f-necas**
Prepare geOrchestra migrations notes

### CODE REVIEW

File: migrations/24.0/README.md [hunk 1]
```diff
@@ -64,3 +64,9 @@ This role allows user to have access to the import tool (datafeeder).
 
 By default, users couldn't use datafeeder application.
 
+
+## Gateway
+
+The `gateway` service has been added to the geOrchestra stack. It replaces the old `proxy` service.
```
Comment (fvanderbiest) [inline migrations/24.0/README.md:70]: Oh my god, no, we have to go through a GIP first ...

**Comment by edevosc2c** [comment]
For georchestra 24.0 I would really like to migrate all the containers to non-root containers.

As this is a new release, it can be a good thing because this is going to involve breaking changes.

**Comment by landryb** [comment]
> For georchestra 24.0 I would really like to migrate all the containers to non-root containers for security reasons.

and i would really like developers to think about the non-containers case ;)

File: migrations/24.0/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+By default the `Area of competence` in console is now disabled.
+
+The functionnality can be enabled in `console.properties` with the line : 
+```
+competenceAreaEnabled=true
+```
+
+## Cas server
+
+Cas server can be upgraded to 6.6.0 and configuration must be updated accordingly.
```
Comment (landryb) [inline migrations/24.0/README.md:83]: 6.6.15 ?

Comment (pmauduit) [reply migrations/24.0/README.md]: indeed: https://github.com/georchestra/georchestra-cas-server/blob/master/gradle.properties#L2

**Pull request closed without merging**